src/Microsoft.AspNet.Mvc.Core/InputFormatter.cs

```diff
@@ -18,6 +18,19 @@ namespace Microsoft.AspNet.Mvc
     public abstract class InputFormatter : IInputFormatter
     {
         /// <summary>
+        /// Returns UTF8 Encoding without BOM and throws on invalid bytes.
+        /// </summary>
+        public static readonly Encoding UTF8EncodingWithoutBOM
+            = new UTF8Encoding(encoderShouldEmitUTF8Identifier: false, throwOnInvalidBytes: true);
+
+        /// <summary>
+        /// Returns UTF16 Encoding which uses littleEndian byte order with BOM and throws on invalid bytes.
+        /// </summary>
+        public static readonly Encoding UTF16EncodingLittleEndian = new UnicodeEncoding(bigEndian: false,
+                                                                                        byteOrderMark: true,
+                                                                                        throwOnInvalidBytes: true);
```

Comment:
Also why not just use the new fields from `OutputFormatter`? Or at least initialize these fields using those?